src/shiv/bootstrap/__init__.py

```diff
@@ -132,10 +132,17 @@ def _first_sitedir_index():
             return index
 
 
-def _extend_python_path(environ, additional_paths):
+def _extend_python_path(environ, sys_path):
+    """Create or extend a PYTHONPATH variable with the frozen environment we are bootstrapping with."""
+
+    # we don't want to clobber any existing PYTHONPATH value, so check for it.
     python_path = environ["PYTHONPATH"].split(os.pathsep) if "PYTHONPATH" in environ else []
-    python_path.extend(additional_paths)
-    environ["PYTHONPATH"] = os.pathsep.join(python_path)
+
+    # extend the existing shiv-manipulated sys.path to include any existing PYTHONPATH
+    sys_path.extend(python_path)
```

Comment:
Would you mind renaming `sys_path` to `sys_path_copy` or even just `paths`, so that future readers of this code are not thinking that this is extending the current run-time's `sys.path` (since you pass `sys.path.copy()` into the function).